software/partdb.yml

```diff
@@ -0,0 +1,14 @@
+name: "Part-DB"
+website_url: "https://docs.part-db.de/"
+source_code_url: "https://github.com/Part-DB/Part-DB-server"
+description: "Part-DB is an Open-Source inventory management system for your electronic components."
```

Comment:
Terms such as open-source should be avoided ([see here](https://github.com/awesome-selfhosted/awesome-selfhosted-data/blob/master/CONTRIBUTING.md#other-guidelines))

```suggestion
description: "An inventory management system for your electronic components."
```